FastSimulation/Calorimetry/interface/CalorimetryManager.h

```diff
@@ -94,6 +96,17 @@ class CalorimetryManager{
 
   void clean(); 
 
+  /** \brief Scale the hits by a scale derived from fast/fullsim comparison
+    * \param particleAtEcalEntrance The particle which's shower should be simulated
+    * \param hitMap Contains the hits for one simulated particle
+    * \return clone of hitMap with scaled energies
+    *
+    * This method finds the corresponding scale as function of the particles
+    * eta at ecal entrance, true energy at ecal entrance, and total simulated energy.
+    * The hits are scaled and returned.
+    */
+  std::map<CaloHitID,float> applyECALScaleFactor( const RawParticle& particleAtEcalEntrance, const std::map<CaloHitID,float>& hitMap ) const;
+
```

Comment:
move all ECAL response code to separate class ECALresponse that resides in 
FastSimulation/Calorimetry (src and interace directories)